test/python/circuit/test_commutation_checker.py

```diff
@@ -433,26 +434,25 @@ def test_cutoff_angles(self, gate_cls):
                 self.assertFalse(scc.commute(generic_gate, [0, 1], [], gate, qargs, []))
 
     @idata(ROTATION_GATES)
-    def test_rotation_mod_2pi(self, gate_cls):
-        """Test the rotations modulo 2pi commute with any gate."""
+    def test_controlled_rotation_mod_4pi(self, gate_cls):
+        """Test the rotations modulo 2pi (controlled 4pi) commute with any gate."""
```

Comment:
Nitpick, maybe we could specify/hint that cp is still 2pi:

```suggestion
        """Test the rotations modulo 2pi (controlled rx, ry, rz -> 4pi) commute with any gate."""
```

Reply:
added in 59dbf24